packaging_assets/package.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
I'd recommend setting some flags for the script to make sure nothing fails silently:
```bash
set -euf -o pipefail
```